cluster/manifests/skipper/deployment.yaml

```diff
@@ -69,6 +69,13 @@ spec:
           - "-metrics-flavour=codahale,prometheus"
           - "-enable-connection-metrics"
           - "-max-audit-body=0"
+          - "-enable-swarm"
+          - "-swarm-port=9990"
+          - "-swarm-label-selector-key=application"
+          - "-swarm-label-selector-value=skipper-ingress"
+          - "-swarm-leave-timeout=5s"
+          - "-swarm-max-msg-buffer=4194304"
+          - "-swarm-namespace=kube-system"
```

Comment:
These flags should be wrapped in `{{ if index .ConfigItems "skipper_clusterratelimit" }}`